sched/sched/sched_setpriority.c

```diff
@@ -69,12 +68,12 @@ static FAR struct tcb_s *nxsched_nexttcb(FAR struct tcb_s *tcb)
    * then use the 'nxttcb' which will probably be the IDLE thread.
    */
 
```

Comment:
I think the above comments `if the affinity mask includes the current CPU` should be modified as well.